subsys/bluetooth/host/long_work.h

```diff
@@ -0,0 +1,20 @@
+/* long_work.h - Workqueue intended for long-running operations. */
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#if defined(CONFIG_BT_LONG_WQ)
+
+void bt_long_work_schedule(struct k_work_delayable *dwork, k_timeout_t timeout);
+void bt_long_work_reschedule(struct k_work_delayable *dwork, k_timeout_t timeout);
+void bt_long_work_submit(struct k_work *dwork);
```

Comment:
The parameter name should probably be just `work` since it's not delayed work in this case.